README.md

```diff
@@ -1,35 +1,56 @@
+<!-- TODO: This is a template SDK README.md.
+It's structured to fit nicely into openfeature.dev, which is configured to include SDK READMEs.
+Complete all the TODOs applicable for your implementation and then create an issue in https://github.com/open-feature/openfeature.dev
+-->
+
 <!-- markdownlint-disable MD033 -->
+<!-- x-hide-in-docs-start -->
 <p align="center">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg">
-    <img align="center" alt="OpenFeature Logo">
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg" />
+    <img align="center" alt="OpenFeature Logo" src="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg" />
   </picture>
 </p>
 
-<h2 align="center">OpenFeature Swift SDK</h2>
+<h2 align="center">OpenFeature <!-- TODO: your language SDK --> SDK</h2>
```

Comment:
```suggestion
<h2 align="center">OpenFeature iOS SDK</h2>
```

I think we talked about calling it the iOS SDK instead of Swift but I could be misremembering.